core/eval.go

```diff
@@ -53,6 +54,38 @@ func evalPING(args []string) []byte {
 	return b
 }
 
+// evalAUTH returns with an encoded "OK" if the user is authenticated
+// If the user is not authenticated, it returns with an encoded error message
+func evalAUTH(args []string, c *Client) []byte {
+	var (
+		b   []byte
+		err error
+	)
+	log.Print("Authenticating user")
```

Comment:
Do not add log statements. We are yet to define logging standards.

Reply:
Removed.